src/App.js

```diff
@@ -26,16 +25,18 @@ function App() {
 
   // our game logic. It is buggy at the time, it seems to be one step behind
   const checkPrice = () => {
-    setNextPrice(formattedCurrency);
-    console.log(formattedCurrency);
-    if (referencePrice > nextPrice && betType === UP_TYPE) {
-      console.log('You loose');
-    } else if (referencePrice < nextPrice && betType === DOWN_TYPE) {
-      console.log('You loose');
-    } else if (referencePrice === nextPrice) {
-      console.log("It's a tie, try again");
+    if (referencePrice > formattedCurrency && betType === UP_TYPE) {
+      console.log('You loose', { referencePrice, formattedCurrency, betType });
```

Comment:
lets remove these logs before merging - we can leave in comments if we like though like:
TODO: add messaging for winning